__tests__/types/challengeBuilder.test.tsx

```diff
@@ -92,7 +93,7 @@ describe("createChallenge", () => {
       .mockRejectedValue(new Error("Firestore error"));
 
     await expect(
-      createChallenge(mockFirestoreCtrl, "Test", "Description", null),
```

Comment:
Same here, what's null ?

Reply:
null here is the location, as this test doesn't use it and mocking the location would require some effort where we don't really need it here.